client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,66 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static targets = ['total', 'label'];
```

Comment:
```suggestion
  static targets = ['label', 'total'];
```

Please sort alpha, it makes it easier to track changes and read the code later.